src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
Not so keen on forcing the array type. Some classes will want to extend the `ObjectRegistry` and offer a quick way of for example passing the class name:

```php
public function load($objectName, $config = []) {
    if (is_string($config)) {
        $config = ['className' => $config];
    }
    ...
}
```